tac/governing-documents/repository-structure.md

```diff
@@ -36,9 +36,6 @@ format suffixes such as `.md`, `.rst`, or `.txt`.
     covers the specifics.
 -   `CONTRIBUTING` \
     Directions on how to contribute code to the project, or a pointer to that information.
--   `CHANGELOG` \
```

Comment:
Can we move this to the optional files section instead of removing it? We can also comment that this file could be a pointer to the release notes available through GitHub.